plugins/coverimage.koplugin/main.lua

```diff
@@ -16,7 +16,17 @@ local T = require("ffi/util").template
 
 local function pathOk(filename)
     local path, name = util.splitFilePathName(filename)
-    return Device:isValidPath(path) and name ~= "" and lfs.attributes(filename, "mode") ~= "directory"
+    if not Device:isValidPath(path) then -- isValidPath expects a trailing slash
+        return false, T(_("Path \"%1\" isn't in a writable location."), path)
+    elseif not util.pathExists(path:gsub("/$", "")) then -- pathExists expects no trailing slash
+        return false, T(_("The path \"%1\" doesn't exist"), path)
```

Comment:
```suggestion
        return false, T(_("The path \"%1\" doesn't exist."), path)
```

Reply:
I like it, too. Thank your for your guidance.